provision/core/mailhog/provision.sh

```diff
@@ -28,20 +28,18 @@ function mailhog_setup() {
     if [[ "aarch64" == $(uname -m) ]]; then
       mhsendmail_bin="https://github.com/evertiro/mhsendmail/releases/download/v0.2.0-M1/mhsendmail_linux_arm64"
     fi
-    if curl --silent -L -o /usr/local/bin/mhsendmail ${mhsendmail_bin}; then
+    if curl --silent -L -o /usr/local/bin/mhsendmail "${mhsendmail_bin}"; then
       chmod +x /usr/local/bin/mhsendmail
       vvv_success " * MHSendmail downloaded"
     else
-      vvv_error " ! MHSendmail failed to download"
+      vvv_error " ! MHSendmail failed to download, check that you have a stable reliable network connection then try again. VVV tried to download ${mhsendmail_bin}"
       return 1
     fi
   fi
 
-  if [[ ! -e /etc/systemd/system/mailhog.service ]]; then
-    vvv_info " * Mailhog service file missing, setting up"
-    # Make it start on reboot
-    cp -f "/srv/provision/core/mailhog/mailhog.service" "/etc/systemd/system/mailhog.service"
-  fi
+  vvv_info " * Adding Mailhog service file"
+  # Make it start on reboot
+  cp -f "/srv/provision/core/mailhog/mailhog.service" "/etc/systemd/system/mailhog.service"
```

Comment:
This without the if would fail on docker right?

Reply:
I mean without copying over the service it'll fail anywhere the service doesn't already exist...

Reply:
the if just checked if the file existed, it didn't check for docker. If we ever changed `mailhog.service` or the file was broken somehow then the check would see it existed and skip copying, and it would never get fixed

Reply:
I meant mostly that there's no service manager in docker mostly.

So the path doesn't exist.